src/Features/Core/Portable/CodeFixes/Configuration/ConfigureSeverity/ConfigureSeverityLevelCodeFixProvider.cs

```diff
@@ -24,11 +24,11 @@ internal sealed partial class ConfigureSeverityLevelCodeFixProvider : IConfigura
     {
         private static readonly ImmutableArray<(string name, string value)> s_editorConfigSeverityStrings =
             ImmutableArray.Create(
-                (nameof(EditorConfigSeverityStrings.None), EditorConfigSeverityStrings.None),
-                (nameof(EditorConfigSeverityStrings.Silent), EditorConfigSeverityStrings.Silent),
-                (nameof(EditorConfigSeverityStrings.Suggestion), EditorConfigSeverityStrings.Suggestion),
-                (nameof(EditorConfigSeverityStrings.Warning), EditorConfigSeverityStrings.Warning),
-                (nameof(EditorConfigSeverityStrings.Error), EditorConfigSeverityStrings.Error));
+                (nameof(EditorConfigSeverityStrings.None), FeaturesResources.None),
```

Comment:
This doesn't seem right -- at least one of these should be the value that goes into the file, right, which is the lowercase strings?

Reply:
Yep, I just notice that case. fixing it now

Reply:
OK, fixed